guides/release/back-end/index.md

```diff
@@ -0,0 +1,74 @@
+Ember developers have great options for how they handle data from
+back end APIs. Ember itself works with any type of back end: REST,
+JSON:API, GraphQL, or anything else. This guide will summarize how
+and where to make API requests.
+Follow the links within it to see examples and learn more.
+
+## How to make API requests
+
+Some common tools for making [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) (create, read, update, delete) requests in Ember include:
+
+- [Ember Data](../models/) is the official data persistence library for Ember. It has a powerful set of tools
+for formatting requests, normalizing responses, and efficiently
+managing a local cache of data. Install `@ember/data` to use it.
+It is included by default in new Ember apps.
+- Native JavaScript methods like [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). Install [`ember-fetch`](https://github.com/ember-cli/ember-fetch) in order to provide support for older browsers, and `import fetch from 'fetch'` to use it.
+- [jQuery Ajax](https://api.jquery.com/jquery.ajax/) requests. Install `@ember/jquery` and set `"jquery-integration": false` in your app's [`optional-features.json`](../configuring-ember/optional-features/) in order to be able to `import jQuery from 'jquery'` in your app.
```

Comment:
Shouldn't it be: _set `"jquery-integration": true` in your app's `optional-features.json`_?

In optional features guide it tells to use Ember CLI to configure a feature. Could be confusing if it recommends one time editing a file directly and another time to use a command.

Reply:
It seems like this is what's recommended later on as well, so it may make sense to update these recommendations to be in sync

Reply:
Agreed, updated to just link to the guide on it